src/index.ts

```diff
@@ -40,6 +41,13 @@ import {
 import { notificationChecker } from './components/notification/checker'
 import { getConfig, setupConfigFromFile } from './components/config'
 
+function getDefaultConfig() {
+  const filesArray = fs.readdirSync('./')
+  const configJsonFile = filesArray.find((x) => x === 'config.json')
+
+  return configJsonFile ? `./${configJsonFile}` : './monika.json'
```

Comment:
shouldn't monika.json take precedence over config.json?

Reply:
With assumption that there are both file in the directory, the sequence is important. So maybe we should cater for that case too..

fixed btw.